src/main/java/edu/cornell/kfs/module/ezra/service/impl/EzraServiceImpl.java

```diff
@@ -258,12 +262,32 @@ protected Award createAward(Proposal proposal, Award oldAward, EzraProposalAward
 		aea.setBudgetEndingDate(ezraAward.getBudgetStopDate());
 		aea.setBudgetTotalAmount(ezraAward.getBudgetAmt());
 
+		Compliance ezraCompliance = getEzraCompliance(ezraAward);
+		if (ObjectUtils.isNotNull(ezraCompliance)) {
+		    boolean everify = Boolean.TRUE.equals(ezraCompliance.getEverify());
+		    aea.setEverify(everify);
+		}
+
 		award.refreshReferenceObject("proposal");
 		award.refreshNonUpdateableReferences();
 		award.setExtension(aea);
 		return award;
 	}
 	
+	protected Compliance getEzraCompliance(EzraProposalAward ezraAward) {
+	    Map<String, Object> criteria = new HashMap<>();
+	    criteria.put(PROJECT_ID_FIELD, ezraAward.getProjectId());
+	    criteria.put(AWARD_PROPOSAL_ID_FIELD, ezraAward.getAwardProposalId());
+	    
+	    Collection<Compliance> ezraCompliances = businessObjectService.findMatching(Compliance.class, criteria);
+	    // There may be duplicate Compliance entries with the same data, but we only need to retrieve one.
```

Comment:
Will those duplicate Compliance entries all contain the same boolean data value for the everify flag?